coalib/misc/Constants.py

```diff
@@ -128,3 +128,5 @@
     r'(?::\d+)?'  # optional port number
     r'(?:/?|[/?]\S+)$',  # path
     re.IGNORECASE)
+
+EXCLUDE_PREFIX = '!'
```

Comment:
I think in this case we should be more precise naming this `SECTION_EXCLUDE_PREFIX` ;)